edx_toggles/toggles/__init__.py

```diff
@@ -9,16 +9,20 @@
 # We create new classes instead of using `LegacyClass = Class` statements
 # for better monitoring of legacy class usage.
 class LegacyWaffleFlag(WaffleFlag):
-    pass
+    def _get_legacy_custom_attribute_name(self):
+        return 'deprecated_compatible_legacy_waffle_class'
 
 
 class LegacyWaffleFlagNamespace(WaffleFlagNamespace):
-    pass
+    def _get_legacy_custom_attribute_name(self):
+        return 'deprecated_compatible_legacy_waffle_class'
```

Comment:
I suggest we use a class attribute instead of an instance method:

```
class LegacyWaffleFlagNamespace(...):
    _LEGACY_CUSTOM_ATTRIBUTE_NAME = 'deprecated_compatible_legacy_waffle_class'
```